prow/hook/server.go

```diff
@@ -229,6 +229,8 @@ func (s *Server) demuxExternal(l *logrus.Entry, externalPlugins []plugins.Extern
 		go func(p plugins.ExternalPlugin) {
 			if err := s.dispatch(p.Endpoint, payload, h); err != nil {
 				l.WithError(err).Errorf("Error dispatching event to external plugin %q.", p.Name)
+			} else {
+				l.WithField("external-plugin", p.Name).Info("Dispatched event to external plugin")
```

Comment:
Add the `external-plugin` log field in the error case as well instead of including the external plugin name in the error message.

Also should this be a `Debug` level log instead of `Info`?  Either makes sense to me.

Reply:
I think I'd like to be able to see this in all cases.